stories/have-a-question.stories.js

```diff
@@ -0,0 +1,30 @@
+// https://github.com/EnCiv/undebate-ssp/issues/115
+
+import React, { useState, useEffect } from 'react'
+
+import HaveAQuestion from '../app/components/have-a-question'
+
+export default {
+    title: 'Have a Question',
+    component: HaveAQuestion,
+    argTypes: {},
+}
+
+const Template = (args, context) => {
+    useEffect(() => {
```

Comment:
for the code below you need onDone so
````
const {onDone}=context
```
onDone is defined in our storybook decorator in .storybook/preview.js, and passed to stories in context.   This isn't exactly what it was for, but will allow us to show the content of the emit message below the component you are testing.  Try it out.